rust/parquet/src/reader/schema.rs

```diff
@@ -177,6 +177,7 @@ impl ParquetTypeConverter {
             PhysicalType::BOOLEAN => Ok(DataType::Boolean),
             PhysicalType::INT32 => self.to_int32(),
             PhysicalType::INT64 => self.to_int64(),
+            PhysicalType::INT96 => self.to_int64(),
```

Comment:
INT96 is for timestamp in nano and when reading to arrow I am converting to timestamp in ms